develop-docs/architecture/index.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Application
+title: Architecture
```

Comment:
```suggestion
title: Application Architecture
```
I think it's easier to quickly grasp "Application/App Architecture" than just "Architecture" by itself.